envFiles/.env.deploy

```
@@ -109,10 +109,18 @@ REDIS_MAPPED_PORT=6379
 FRONTEND_URL="http://localhost:4321"
 
 ##### OTEL and Observability #####
-API_OTEL_ENABLED=true # make it "false" if don't want span logging
-API_OTEL_SERVICE_NAME="talawa-api"
-API_OTEL_ENVIRONMENT="production"
-API_OTEL_SAMPLING_RATIO="1"
-API_OTEL_EXPORTER_OTLP_ENDPOINT="http://localhost:4318/v1/traces" #TODO: after integrating Observability tool
+API_OTEL_ENABLED=true
+API_OTEL_ENVIRONMENT=production
+API_OTEL_EXPORTER_OTLP_ENDPOINT="http://jaeger:4318/v1/traces" # Container-to-container
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)